### PR TITLE
Bug fix collection brick content offset calculation

### PR DESCRIPTION
**Pull request by wfsttam**

### CODE REVIEW

**Comment by thevwu** [comment]
preferredLayoutAttributesFittingAttributes was returning bounds.height && bounds.width of 0 for cells that were already sized, causing an infinite loop.

File: Tests/Bricks/CollectionBrickTests.swift [hunk 1]
```diff
@@ -129,6 +129,43 @@ class CollectionBrickTests: XCTestCase {
...
+
+        let collectionSection = BrickSection("CollectionSection", bricks: [
+            DummyBrick("1", height: .Auto(estimate: .Fixed(size: 1000))),
+            DummyBrick("2", height: .Auto(estimate: .Fixed(size: 1000))),
+            DummyBrick("3", height: .Auto(estimate: .Fixed(size: 1000))),
+            DummyBrick("4", height: .Auto(estimate: .Fixed(size: 1000)))
+            ])
+
+        let section = BrickSection("RootSection", bricks: [
+            CollectionBrick("Collection1", dataSource: CollectionBrickCellModel(section: collectionSection, configureHandler: { cell in
```
Comment (rubencagnie) [inline Tests/Bricks/CollectionBrickTests.swift]: You can use the `brickTypes: [DummyBrick.self])` in the initializer to make the constructor easier readable

File: Tests/Bricks/CollectionBrickTests.swift [hunk 2]
```diff
@@ -129,6 +129,43 @@ class CollectionBrickTests: XCTestCase {
...
+                cell.brickCollectionView.registerBrickClass(DummyBrick.self)
+            }))
+            ])
+        brickView.setSection(section)
```
Comment (rubencagnie) [inline Tests/Bricks/CollectionBrickTests.swift]: Use `setupSectionAndLayout`

File: Tests/Bricks/CollectionBrickTests.swift [hunk 3]
```diff
@@ -129,6 +129,43 @@ class CollectionBrickTests: XCTestCase {
...
+        brickView.layoutSubviews()
+
+        let expectation = expectationWithDescription("Wait for batch updates")
+        brickView.performBatchUpdates({
```
Comment (rubencagnie) [inline Tests/Bricks/CollectionBrickTests.swift]: Why is this needed?

File: Tests/Bricks/CollectionBrickTests.swift (hunk 1 above)
Comment (wfsttam) [inline Tests/Bricks/CollectionBrickTests.swift]: fixed

File: Tests/Bricks/CollectionBrickTests.swift (hunk 2 above)
Comment (wfsttam) [inline Tests/Bricks/CollectionBrickTests.swift]: fixed

File: Tests/Bricks/CollectionBrickTests.swift (hunk 3 above)
Comment (wfsttam) [inline Tests/Bricks/CollectionBrickTests.swift]: removed
